docker-compose.yml

```diff
@@ -95,4 +95,5 @@ networks:
 volumes:
   api-db-data:
   frontend-node-cache:
+    name: pims-frontend-node-cache
```

Comment:
The default behaviour in docker-compose is to prefix volumes with the folder name where the code was downloaded. Since each developer can put the code wherever he/she wants it can be difficult to find the name of the docker volumes that are created in a `docker-compose.yml` file.

Recent versions of `docker-compose` added this new option **name** where the name is hard-coded and not prefixed. This was added here so the `Makefile` can delete the node_modules cache when you run `make clean`

Reply:
I like this